### PR TITLE
COVID flexibility

### PR DESCRIPTION
**Pull request by awisema**
For review, but marked draft until we learn from Jordyn if she wants to hold off deployment.

### CODE REVIEW

File: app/models/health/participation_form.rb [hunk 1]
```diff
@@ -35,6 +35,7 @@ class ParticipationForm < HealthBase
     scope :valid, -> do
       parent_ids = Health::ParticipationFormFile.where.not(parent_id: nil).select(:parent_id)
       where.not(location: [nil, '']).
```
Comment (eanders) [inline app/models/health/participation_form.rb:37]: Is this a problem?  Didn't we remove the location field from the form?

File: app/models/health/qualifying_activity.rb [hunk 2]
```diff
@@ -395,11 +395,26 @@ def outreach?
     end
 
     def modifiers
+      TodoOrDie('Remove MH COVID flexibility', by: '2023-01-01')
+
       modifiers = []
+      case activity.to_sym
+      when :cha, :discharge_follow_up
+        if [:phone_call, :video_call].include?(mode_of_contact.to_sym)
+          contact_modifier = self.class.modes_of_contact[:in_person].try(:[], :code)
```
Comment (eanders) [inline app/models/health/qualifying_activity.rb]: Why is this a `try`?  It doesn't look like anything here should be changing, and it's referencing a class method, which should be relatively fixed right?

**Review by eanders — APPROVED** [review]
Looks good.  A couple of questions in-line, and waiting on tests to pass.

File: app/models/health/participation_form.rb (hunk 1 above)
Comment (awisema) [reply app/models/health/participation_form.rb:37]: We removed the location field from the UI, but any forms that were submitted with a location value, we left as they were... So, this deals with the legacy case.

File: app/models/health/qualifying_activity.rb (hunk 2 above)
Comment (awisema) [reply app/models/health/qualifying_activity.rb]: Dead code left over from copy-paste :-)

**Pull request merged**